src/scss/base/_layout.scss

```diff
@@ -2,6 +2,7 @@
 @use '../settings/variables' as v;
 @use '../settings/include-media' as *;
 @use '../tools/mixins/type';
+@use '../tools/functions/spacing' as spacing;
```

Comment:
can probably remove the 'as' from this too :)

Reply:
Will be creating a ticket to tidy this all up. Once we've agreed on a convention.

Reply:
I'm not 100% sure if it'll work, but would it be possible to potentially do:

```scss
@use '../tools/functions/index as functions;
```

and then reference `functions.spacing()`

I think it should work with that syntax and make the namespaces a bit cleaner?

Reply:
I can give this a try.

Reply:
This is now fixed here https://github.com/justeat/fozzie/pull/401. I took the function route it worked well 👍